bin/main.rb

```diff
@@ -1,2 +1,102 @@
 #!/usr/bin/env ruby
-puts 'Hello World'
\ No newline at end of file
+
+puts 'Welcome to Tic Tac Toe game.'
+puts '\n\n'
+puts '----------------Instructions------------'
+puts 'The game require two players'
+puts 'Player can choose number 1 to 9 to represent a move    '
+puts 'But a number(move) can only be played once'
+puts ''
+
+class Player
+  attr_accessor :player_x
+
+  attr_accessor :player_o
+
+  def initialize
+    # Collect names if the players
+    puts 'Player 1, Enter Name:'
+    @player_x = gets.chomp
+    puts 'Player 2, Enter Name:'
+    @player_o = gets.chomp
+    # Show the name of the player and the symbol
+    puts "#{@player_x} will be X AND #{@player_o} will be O"
+  end
+end
+
+class Board
+  attr_reader :board
+
+  # Show what the Tic Tac Toe Board look like
+  def initialize
+    puts 'This is the board of Tic Tac Toe '
+    puts '  ___ ___ ___  '
+    puts ' | 1 | 2 | 3 | '
+    puts ' |___|___|___| '
+    puts ' | 4 | 5 | 6 | '
+    puts ' |___|___|___| '
+    puts ' | 7 | 8 | 9 | '
+    puts ' |___|___|___| '
+    puts '               '
+
+    @board = [' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ']
+  end
+
+  # Display board after a player has played
+  def display_board(board)
+    puts "#{board[0]} | #{board[1]} | #{board[2]}"
+    puts '---------'
+    puts "#{board[3]} | #{board[4]} | #{board[5]}"
+    puts '---------'
+    puts "#{board[6]} | #{board[7]} | #{board[8]}"
+  end
+end
+
+class Game
+  # Start a new game by geting player names
+  def initialize
+    @player = Player.new
+    @board = Board.new
+  end
+
+  def play
+    moves = 0
+    while moves < 9
```

Comment:
* [ ] [REQUIRED]  Using out-of-nowhere numbers to terminate a loop (for example, `5.times do ... end`) is a signal that something is wrong. Please use a terminating variable instead. See below for an example where `game_on` is the terminating variable:
```
game_on = true

# initial instructions for player

while game_on
  # loop for each move

  if winner # and/or draw (the exact condition in this milestone is not important)
    game_on = false
  end
end
```
```